src/librustdoc/lib.rs

```diff
@@ -247,14 +250,16 @@ pub fn main_args(args: &[String]) -> isize {
 
     if let Some(ref p) = css_file_extension {
         if !p.is_file() {
-            println!("{}", "--extend-css option must take a css file as input");
+            writeln!(
```

Comment:
Because the prompting for `--help` should only be used for error in the invocation syntax, such as missing operands or missing arguments.